templates/elasticsearch_exporter.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Prometheus Exporter for ElasticSearch
+Documentation=https://github.com/justwatchcom/elasticsearch_exporter/README.md
+After=network.target
+#After=elasticsearch.service network.target
+#Requires=elasticsearch.service
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=simple
+Restart=always
+ExecStart={{elasticsearch_exporter_install_path}}/prometheus_elasticsearch_exporter/elasticsearch_exporter
+EnvironmentFile=-/etc/default/elasticsearch_exporter
+User={{elasticsearch_exporter_user}}
+Group={{elasticsearch_exporter_group}}
```

Comment:
Yeah, totally! :+1:

Reply:
well actually this is not needed at all, because systemd already takes care of everything when `Type=simple`, I completely removed the variable since it's useless